package.json

```diff
@@ -20,6 +20,7 @@
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
 		"react-router-dom": "^6.21.3",
+		"styled-components": "^6.1.8",
```

Comment:
Can be removed if not used.

Reply:
done. [commit](https://github.com/dscnitrourkela/project-udon/pull/17/commits/25b1a1a1ed993d4a08e4fba0e90c16de2898e65d)